Source/ModelGroup.cs

```diff
@@ -246,6 +272,7 @@ internal class Model
         public Manifest.Camera Camera = null;
         public bool Animated = false;
         public bool? Loadable = true;
+        public bool Individual = false;
```

Comment:
```suggestion
        public bool SeparateBuffers = false;
```

I would recommend this name.